src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+import { LabeledValue } from "antd/lib/select";
+
+export type Project = { identifier: number; name: string };
+
+export interface ProjectSelectProps extends SelectProps {
+  projects: Project[];
```

Comment:
I though you had decided to go with `ProjectMinimal` here?

Reply:
Please see 76ba8665596da5cb030453f0907409efa334b225.

Reply:
After generalising the component, I think picking id and name form IridaBase makes the most sense now. Please see 9942971a7205097b512877c8fa6c2c6830f0b047.